site/_posts/2017-03-20-yet-another-move-by-euwe.md

```diff
@@ -0,0 +1,94 @@
+---
+title: "Last Week in ManageIQ: Yet Another move by Euwe"
+author: Libor Pichler
```

Comment:
Please use github username as author, so when we implement linking to author it'll be easier!